### PR TITLE
Adding my Planck Keymap

### PR DESCRIPTION
**Pull request by eshesh2**
## Description

Adding my Planck keymap to QMK

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [X] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/planck/keymaps/eshesh2/config.h [hunk 1]
```diff
@@ -0,0 +1,41 @@
+#pragma once
```
Comment (drashna) [inline keyboards/planck/keymaps/eshesh2/config.h:1]: Could you add a GPL2+ compatible license header here? 

For instance: 
https://github.com/qmk/qmk_firmware/blob/28ff51175b6d45ce5a8dbfe7ed6e7a12df3bc8a8/quantum/template/base/keyboard.c#L1-L15

**Review by drashna — CHANGES_REQUESTED** [review]

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
+!eshesh2's Planck Layout [https://i.imgur.com/eVgHH6k.png]
+
+# eshesh2's Planck layout with a rotary encoder on the top left of the Planck. It is designed for everyday use along with a numpad layer for quickly typing in Excel and other workbook programs.
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:3]:
```suggestion
![eshesh2's Planck Layout](https://i.imgur.com/eVgHH6k.png)

eshesh2's Planck layout with a rotary encoder on the top left of the Planck. It is designed for everyday use along with a numpad layer for quickly typing in Excel and other workbook programs.
```

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+# Qwerty
+ * ,-----------------------------------------------------------------------------------.
+ * | Mute |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Tab  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Esc  | Ctrl | GUI  | Alt  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
+ * `-----------------------------------------------------------------------------------'
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:14]:
```suggestion

    ,-----------------------------------------------------------------------------------.
    | Mute |   Q  |   W  |   E  |   R  |   T  |   Y  |   U  |   I  |   O  |   P  | Bksp |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    | Tab  |   A  |   S  |   D  |   F  |   G  |   H  |   J  |   K  |   L  |   ;  |  "   |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    | Shift|   Z  |   X  |   C  |   V  |   B  |   N  |   M  |   ,  |   .  |   /  |Enter |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    | Esc  | Ctrl | GUI  | Alt  |Lower |    Space    |Raise | Left | Down |  Up  |Right |
    `-----------------------------------------------------------------------------------'
```

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
+!eshesh2's Planck Layout [https://i.imgur.com/eVgHH6k.png]
+
+# eshesh2's Planck layout with a rotary encoder on the top left of the Planck. It is designed for everyday use along with a numpad layer for quickly typing in Excel and other workbook programs.
+
+# Qwerty
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:5]:
```suggestion
## Qwerty
```

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+ # Lower
+ * ,-----------------------------------------------------------------------------------.
+ * | Play |   !  |   @  |   #  |   $  |   %  |   ^  |   &  |   *  |   (  |   )  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   _  |   +  |   {  |   }  |  |   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO ~ |ISO | | Home | End  |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+ * `-----------------------------------------------------------------------------------'
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:25]:
```suggestion
## Lower

    ,-----------------------------------------------------------------------------------.
    | Play |   !  |   @  |   #  |   $  |   %  |   ^  |   &  |   *  |   (  |   )  | Bksp |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   _  |   +  |   {  |   }  |  |   |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO ~ |ISO | | Home | End  |      |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
    `-----------------------------------------------------------------------------------'
```

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 6]
```diff
@@ -0,0 +1,60 @@
...
+# Raise
+ * ,-----------------------------------------------------------------------------------.
+ * |   `  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Bksp |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   -  |   =  |   [  |   ]  |  \   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO # |ISO / |Pg Up |Pg Dn |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
+ * `-----------------------------------------------------------------------------------'
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:36]:
```suggestion
## Raise

    ,-----------------------------------------------------------------------------------.
    |   `  |   1  |   2  |   3  |   4  |   5  |   6  |   7  |   8  |   9  |   0  | Bksp |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    | Del  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6  |   -  |   =  |   [  |   ]  |  \   |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |  F7  |  F8  |  F9  |  F10 |  F11 |  F12 |ISO # |ISO / |Pg Up |Pg Dn |      |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |      |      |      |      |             |      | Next | Vol- | Vol+ | Play |
    `-----------------------------------------------------------------------------------'
```

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+ # Numpad layer
+ * ,-----------------------------------------------------------------------------------.
+ * |      |      |  Up  |      |      |      |      |      |NumLck|   7  |  8   |  9   |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      | Left | Down | Right|      |      |      |      |      |   4  |  5   |   6  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |Enter |  Tab |      |      |      |      |      |      |   1  |  2   |   3  |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * | Exit |      |      |      |      |             |      |      |   .  |  0   |Bksp |
+ * `-----------------------------------------------------------------------------------'
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:47]:
```suggestion
# Numpad layer

    ,-----------------------------------------------------------------------------------.
    |      |      |  Up  |      |      |      |      |      |NumLck|   7  |  8   |  9   |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      | Left | Down | Right|      |      |      |      |      |   4  |  5   |   6  |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |Enter |  Tab |      |      |      |      |      |      |   1  |  2   |   3  |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    | Exit |      |      |      |      |             |      |      |   .  |  0   |Bksp |
    `-----------------------------------------------------------------------------------'
```

File: keyboards/planck/keymaps/eshesh2/readme.md [hunk 8]
```diff
@@ -0,0 +1,60 @@
...
+ * ,-----------------------------------------------------------------------------------.
+ * |      | Reset|Debug | RGB  |RGBMOD| HUE+ | HUE- | SAT+ | SAT- |BRGTH+|BRGTH-|  Del |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |MUSmod|Aud on|Audoff|AGnorm|AGswap|Qwerty|      |      |Numpad|      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |Voice-|Voice+|Mus on|Musoff|MIDIon|MIDIof|TermOn|TermOf|      |      |      |
+ * |------+------+------+------+------+------+------+------+------+------+------+------|
+ * |      |      |      |      |      |             |      |      |      |      |      |
+ * `-----------------------------------------------------------------------------------'
+ */
```
Comment (fauxpark) [inline keyboards/planck/keymaps/eshesh2/readme.md:60]:
```suggestion
# Adjust (Lower + Raise)

                         v------------------------RGB CONTROL--------------------v
    ,-----------------------------------------------------------------------------------.
    |      | Reset|Debug | RGB  |RGBMOD| HUE+ | HUE- | SAT+ | SAT- |BRGTH+|BRGTH-|  Del |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |      |MUSmod|Aud on|Audoff|AGnorm|AGswap|Qwerty|      |      |Numpad|      |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |Voice-|Voice+|Mus on|Musoff|MIDIon|MIDIof|TermOn|TermOf|      |      |      |
    |------+------+------+------+------+------+------+------+------+------+------+------|
    |      |      |      |      |      |             |      |      |      |      |      |
    `-----------------------------------------------------------------------------------'
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by drashna — APPROVED** [review]

**Review by fauxpark — APPROVED** [review]

**Pull request merged**